cfg/DB/table/logs.ddl

```diff
@@ -27,4 +27,12 @@ alter table logs
 -- alter table logs
 --     add constraint logs_vacancy_id_fk
 --         foreign key (vacancy_id) references vacancies
---             on update cascade on delete cascade;
\ No newline at end of file
+--             on update cascade on delete cascade;
+
+update logs
+set action_code=108
+where logs.text LIKE '% назначил %';
```

Comment:
why not accord with 'kod_deystviya'?

Reply:
its an refactor for old table, we need to change old logs to use with new features. there are new codes for send cv and interview. Earlier all these actions were recorded with code perform (as far as I remember)